packages/orbit-design-tokens/src/__tests__/tokensToCssVars.ts

```diff
@@ -0,0 +1,45 @@
+import transformToCssVars from "../utils/tokensToCssVars";
+
+const tokens = {
+  paletteProductNormal: "#ff0000",
+  paletteProductNormalHover: "#e50000",
+  paletteProductNormalActive: "#cc0000",
+};
+
+describe("transformToCssVars", () => {
+  it("should convert object into css variables string", () => {
+    expect(transformToCssVars({ tokens, cssClass: false })).toMatchInlineSnapshot(`
+      "--palette-product-normal: #ff0000;
+      --palette-product-normal-hover: #e50000;
+      --palette-product-normal-active: #cc0000;
+      "
+    `);
+  });
+
+  it("should convert object into css variables as a class", () => {
+    expect(transformToCssVars({ tokens })).toMatchInlineSnapshot(`
+      ":root {
+        --palette-product-normal: #ff0000;
+        --palette-product-normal-hover: #e50000;
+        --palette-product-normal-active: #cc0000;
+
+      }"
+    `);
+  });
+
+  it("should convert nested object", () => {
```

Comment:
do we really need this?

Reply:
it's might be needed, palette is nested object